### PR TITLE
Fix conversion for urls containing quotes or parentheses

### PR DESCRIPTION
**Pull request by alin23**

### CODE REVIEW

File: static_precompiler/utils.py [hunk 1]
```diff
@@ -130,26 +130,32 @@ def run_command(args, input=None, cwd=None):
...
 
     @staticmethod
     def convert_url(url, source_dir):
         assert source_dir[-1] == "/"
-        url = url.strip(' \'"')
+
         if url.startswith(('http://', 'https://', '/', 'data:')):
             return url
-        return urljoin(settings.STATIC_URL, urljoin(source_dir, url))
+        return '{0}'.format(urljoin(settings.STATIC_URL, urljoin(source_dir, url)))
```
Comment (andreyfedoseev) [inline static_precompiler/utils.py]: Why?

Comment (alin23) [reply static_precompiler/utils.py]: Oh sorry, that's a leftover from a previous try. I'm removing it right now

**Comment by alin23** [comment]
Hi @andreyfedoseev .

Is there anything more I should do here before it can be merged?

**Comment by andreyfedoseev** [comment]
@alin23 

Two things:

1. Please limit the scope of the pull request. Changes to the CoffeeScript compiler don't belong here.
2. Could you provide a sample CSS that doesn't work as expected with the existing URL converter and describe how you'd expect it to handle that case?

**Comment by alin23** [comment]
@andreyfedoseev 
I removed the Coffeescript commit.

Here is a sample css that makes the current url converter fail:
```css
.custom-checkbox .custom-control-input:checked ~ .custom-control-indicator {
  background-image: url("data:image/svg+xml;charset=utf8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 8 8'%3E%3Cpath fill='rgba(255, 255, 255, 0.97)' d='M6.564.75l-3.59 3.612-1.538-1.55L0 4.26 2.974 7.25 8 2.193z'/%3E%3C/svg%3E"); }
```

This is from a Bootstrap 4 SASS compiled to CSS.

**Comment by andreyfedoseev** [comment]
I see.

How about this?

```
def convert_url(self, url):
    url = url.strip()
    if not url:
        return url
    original_quote = url[0] if url[0] in ('"', "'") else "'"
    url = url.strip("\"'")
    ....
    return "{original_quote}{url}{original_quote}".format(original_quote=original_quote, url=url)
```

Then in `def convert` don't add the enclosing single quotes to the converted url.

**Comment by alin23** [comment]
@andreyfedoseev your solution looks better. Can you please check to see if I didn't miss anything?

**Pull request merged**